dark_evaluation.py

```diff
@@ -1,23 +1,43 @@
-from digicampipe.calib.camera import filter
+#!/usr/bin/env python
+'''
+Extract dark.npz and bias_curve_dark.npz from observations.
+
+    Example call:
+    dark_evaluation.py . path/to/SST1M01_20171030.002.fits.fz
+
+Usage:
+  dark_evaluation.py <output_directory> <files>...
+
+Options:
+  -h --help     Show this screen.
+'''
+from digicampipe.calib.camera import filter, r0
 from digicampipe.io.event_stream import event_stream
 from digicampipe.io.save_adc import save_dark
 from digicampipe.io.save_bias_curve import save_bias_curve
 from cts_core.camera import Camera
 from digicampipe.utils import geometry
 import matplotlib.pyplot as plt
 import numpy as np
+from tqdm import tqdm
+import pkg_resources
+from os import path
+from docopt import docopt
 
+def main(output_directory, files):
+    camera_config_file = pkg_resources.resource_filename(
+        'digicampipe',
+        path.join(
+            'tests',
+            'resources',
+            'camera_config.cfg'
+        )
+    )
 
-if __name__ == '__main__':
-    # Data configuration
+    dark_file_path = path.join(output_directory, 'dark.npz')
+    dark_trigger_file_path = path.join(output_directory, 'bias_curve_dark.npz')
 
-    directory = '/home/alispach/data/CRAB_01/'
-    filename = directory + 'CRAB_01_0_000.%03d.fits.fz'
-    file_list = [filename % number for number in [2]]
-    camera_config_file = '/home/alispach/ctasoft/CTS/config/camera_config.cfg'
-    dark_filename = 'dark.npz'
 
-    trigger_filename = 'bias_curve_dark.npz'
     thresholds = np.arange(0, 400, 10)
     unwanted_patch = [306, 318, 330, 342, 200]
```

Comment:
actually this is also supposed to be an option. it was the patches that were not working during the Crab observations.
I suppose we can live it like this since all the data we have is containing this problem.